modules/metrics/pom.xml

```diff
@@ -85,6 +85,13 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+
+        <dependency>
```

Comment:
Please reflect this change in Gradle file as well

Reply:
Fix it